feature-libs/cart/base/core/facade/multi-cart.service.ts

```diff
@@ -334,6 +334,18 @@ export class MultiCartService implements MultiCartFacade {
     );
   }
 
+  /**
+   * Remove Cart
+   *
+   * Used for cleaning cart in local state, when we get information that it no longer exists in the backend.
+   * For removing particular cart in both places use DeleteCart actions.
+   *
+   * @param cartId
+   */
```

Comment:
There's already a jsdoc in the abstract class, which will be shown in an IDE.
Therefore, we don't need it here.

Reply:
I can remove it, but then it will look weird as other exposed functions in this service has them as well

do you want me to remove all the rest too instead?

Reply:
removed as discussed~